common/export-common-cartridge-util/src/java/org/sakaiproject/exporter/util/ForumsExport.java

```diff
@@ -89,6 +90,9 @@ public boolean outputAllForums(ZipPrintStream out) throws IOException, TypeExcep
 				if (text != null) {
 					text = FormattedText.convertPlaintextToFormattedText(text);
```

Comment:
Although this escapes all the HTML entities in the content you will still end up with unescaped `<br>`s for newlines in the output. I'm guessing this is OK, but when you have a HTML description all HTML is escaped.